tests/unit/BaseIndexTest.php

```diff
@@ -152,14 +152,35 @@ public function testFacetFields()
         /** @var ArrayList $parents */
         $query->addFacetFilter('Parent', $id);
         $result = $index->buildSolrQuery($query);
-        $filterQuery = $result->getFilterQuery('facets');
+        $filterQuery = $result->getFilterQuery('andFacets');
         $this->assertEquals('SiteTree_ParentID:' . $id, $filterQuery->getQuery());
         $query->addFacetFilter('Parent', 5);
-        $filterQuery = $index->buildSolrQuery($query)->getFilterQuery('facets');
+        $filterQuery = $index->buildSolrQuery($query)->getFilterQuery('andFacets');
         $this->assertContains('SiteTree_ParentID:5 AND SiteTree_ParentID:' . $id, $filterQuery->getQuery());
+        $index->addFacetField(Page::class, ['BaseClass' => SiteTree::class, 'Field' => 'ParentID', 'Title' => 'Child']);
+        $query->addOrFacetFilter('Parent', 3);
+        $query->addOrFacetFilter('Child', 5);
```

Comment:
@Firesphere Great work buddy. 🐮 
```suggestion
        $query->addOrFacetFilter('Parent', 5);
```

Reply:
That would merge it in to the Parent, causing it to become `Parent:3 AND Parent:5` instead of `Parent:3 OR Child:5`

The goal is to have the groups still be AND, but each group to be separated in to OR

Reply:
That would merge it in to the Parent, causing it to become `Parent:3 AND Parent:5` instead of `Parent:3 OR Child:5`

The goal is to have the groups still be AND, but each group to be separated in to OR